wit/bindgen/generator.go

```diff
@@ -808,6 +808,13 @@ func (g *generator) flagsRep(file *gen.File, dir wit.Direction, flags *wit.Flags
 
 func (g *generator) enumRep(file *gen.File, dir wit.Direction, e *wit.Enum, goName string) string {
 	var b strings.Builder
+
+	// builder for enum reverse lookup
+	var rb strings.Builder
```

Comment:
Not sure you need to use a separate builder; just iterate again.

Reply:
hmm I tried it, but it seems like DeclareName returns a different value when called again. So I ended up using a second builder. I will try again to be sure.

Reply:
an option could be to create a temporary map to store wit-string-name to value returned by DeclareName and then use that in the second iteration. 

having second iteration seems like a cleaner approach to me as well.

Reply:
You don't need the Go names to declare the string values.

Reply:
This doesn't need a separate builder at all.

Reply:
oh that is right. I need more coffee :)